Wikibase.php

```diff
@@ -269,6 +264,8 @@
 	$wgWBRepoSettings['useKartographerGlobeCoordinateFormatter'] = true;
 	// Data-Bridge
 	$wgWBRepoSettings['dataBridgeEnabled'] = true;
+	$wgWBRepoSettings['allowDataAccessInUserLanguage'] = true;
+	$wgWBRepoSettings['entityAccessLimit'] = '500';
```

Comment:
Should this be an integer?

Reply:
> Should this be an integer?

Yeah, it is, but I probably didn't insert it correctly? but it's an integer.

Reply:
Remove the quotations, otherwise it is a string.

Reply:
> Remove the quotations, otherwise it is a string.

ah, done, thanks.